src/puppetlabs/puppetdb/config.clj

```diff
@@ -318,8 +318,6 @@
   "Configures the global properties from the user defined config"
   [{:keys [global] :as config}]
   (let [product-name (normalize-product-name (get global :product-name "puppetdb"))]
-    (when (:event-query-limit global)
```

Comment:
Excellent point - sorry it took so long for me to get around to making such a trivial change.